deployment/CONFIGURATION.md

```diff
@@ -0,0 +1,68 @@
+# POA Token Bridge / Deployment Configuration
```

Comment:
We should be consistent throughout the repo, and rename Token Bridge / Token bridge to TokenBridge.